colcon_core/shell/template/package.sh.em

```diff
@@ -19,6 +19,10 @@ colcon_prepend_unique_value() {
   IFS=":"
   # start with the new value
   _all_values="$_value"
+  # wordaround SH_WORD_SPLIT not being set in zsh
```

Comment:
word->work

Reply:
Good catch. Fixed in 7c0676b89edf35c7df7842ca0581927c01e2dc67.